src/css/custom.css

```diff
@@ -39,6 +39,7 @@
   --ifm-color-info: #1976d2;
   --ifm-color-warning: #ef6c00;
   --ifm-color-danger: #d84315;
+
```

Comment:
```suggestion
```

Can we also remove the other formatting changes? I think we don't currently have an auto-formatter for the React bits so pre-commit is not catching it. We should add this in an another PR. Thanks!

Reply:
@jaimergp I will try to remove the formatting changes. I have tried to pay attention not use the keyboard shortcut to auto format the code but it is such a reflex.